plugins/modules/mongodb_user.py

```diff
@@ -355,6 +370,10 @@ def main():
             if db_name != "admin":
                 module.fail_json(msg='The localhost login exception only allows the first admin account to be created')
             # else: this has to be the first admin user added
+            use_localhost_exception = True
+        else:
+            # Is anyone really using pymongo older than 3.0?
+            use_localhost_exception = True
```

Comment:
Once #179 is merged, this will also need to make sure that `auth_mechanism` is not defined.

Reply:
Mmm, I'm not so sure about that. From what I gather on https://github.com/ansible-collections/community.mongodb/issues/177 it would be needed for that use case. Perhaps others too?

Reply:
Yeah. I dropped this and went back to only checking if login_user is None.